src/app/features/info/about/about.component.ts

```diff
@@ -9,49 +9,10 @@ export class AboutComponent implements OnInit {
 
   constructor() { }
 
-  body: any
+  body: any;
 
   ngOnInit() {
-    this.body = `
-      # Binary Coffee
-
-      Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
-      Doloremque veniam possimus impedit. Id perferendis iste
-      consequuntur iusto consectetur rerum soluta nostrum tenetur laboriosam quia,
-      sapiente, aut voluptates, non accusantium ratione!
-      Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
-      Doloremque veniam possimus impedit. Id perferendis iste
-      consequuntur iusto consectetur rerum soluta nostrum tenetur laboriosam quia,
-      sapiente, aut voluptates, non accusantium ratione!
-      Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
-      Doloremque veniam possimus impedit. Id perferendis iste
-      consequuntur iusto consectetur rerum soluta nostrum tenetur laboriosam quia,
-      sapiente, aut voluptates, non accusantium ratione!
-      Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
-      Doloremque veniam possimus impedit. Id perferendis iste
-      consequuntur iusto consectetur rerum soluta nostrum tenetur laboriosam quia,
-      sapiente, aut voluptates, non accusantium ratione!
-
-      # DC's Community
-
-      Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
-      Doloremque veniam possimus impedit. Id perferendis iste
-      consequuntur iusto consectetur rerum soluta nostrum tenetur laboriosam quia,
-      sapiente, aut voluptates, non accusantium ratione!
-      Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
-      Doloremque veniam possimus impedit. Id perferendis iste
-      consequuntur iusto consectetur rerum soluta nostrum tenetur laboriosam quia,
-      sapiente, aut voluptates, non accusantium ratione!
-      Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
-      Doloremque veniam possimus impedit. Id perferendis iste
-      consequuntur iusto consectetur rerum soluta nostrum tenetur laboriosam quia,
-      sapiente, aut voluptates, non accusantium ratione!
-      Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
-      Doloremque veniam possimus impedit. Id perferendis iste
-      consequuntur iusto consectetur rerum soluta nostrum tenetur laboriosam quia,
-      sapiente, aut voluptates, non accusantium ratione!
-
-    `;
+    this.body = '';
```

Comment:
I think this file with the text that we are going to show, should be in the assets